src/R/Wpf/Impl/Brushes.cs

```diff
@@ -16,31 +18,32 @@ public static class Brushes {
-        public static object HeaderBackground { get; set; } = SystemColors.WindowBrushKey;
+        public static object DetailPaneBackgroundKey { get; set; } = SystemColors.WindowBrushKey;
+        public static object GrayTextBrushKey { get; set; } = SystemColors.GrayTextBrushKey;
+        public static object HeaderBackgroundKey { get; set; } = SystemColors.WindowBrushKey;
         public static object HeaderColorsDefaultBrushKey { get; set; } = SystemColors.WindowBrushKey;
         public static object HeaderColorsDefaultTextBrushKey { get; set; } = SystemColors.WindowTextBrushKey;
         public static object HeaderColorsMouseDownBrushKey { get; set; } = SystemColors.WindowBrushKey;
         public static object HeaderColorsMouseDownTextBrushKey { get; set; } = SystemColors.WindowTextBrushKey;
         public static object HeaderColorsMouseOverBrushKey { get; set; } = SystemColors.WindowBrushKey;
         public static object HeaderColorsMouseOverTextBrushKey { get; set; } = SystemColors.WindowTextBrushKey;
         public static object HeaderColorsSeparatorLineBrushKey { get; set; } = SystemColors.ActiveBorderBrushKey;
-        public static object IndicatorFillBrushKey { get; set; } = SystemColors.WindowFrameColor;
+        public static object IndicatorFillBrushKey { get; set; } = SystemColors.WindowFrameColorKey;
         public static object InfoBackgroundKey { get; set; } = SystemColors.InfoBrushKey;
         public static object InfoTextKey { get; set; } = SystemColors.InfoTextBrushKey;
-        public static object LegalMessageBackground { get; set; } = SystemColors.ControlBrushKey;
-        public static object ListPaneBackground { get; set; } = SystemColors.WindowBrushKey;
+        public static object LegalMessageBackgroundKey { get; set; } = SystemColors.ControlBrushKey;
+        public static object ListPaneBackgroundKey { get; set; } = SystemColors.WindowBrushKey;
         public static object SplitterBackgroundKey { get; set; } = SystemColors.WindowBrushKey;
         public static object ToolWindowBorderKey { get; set; } = SystemColors.WindowBrushKey;
         public static object ToolWindowButtonDownBorderKey { get; set; } = SystemColors.WindowBrushKey;
         public static object ToolWindowButtonDownKey { get; set; } = SystemColors.WindowBrushKey;
         public static object ToolWindowButtonHoverActiveBorderKey { get; set; } = SystemColors.WindowBrushKey;
         public static object ToolWindowButtonHoverActiveKey { get; set; } = SystemColors.WindowBrushKey;
-        public static object UIText { get; set; } = SystemColors.ControlTextBrushKey;
+        public static object UITextKey { get; set; } = SystemColors.ControlTextBrushKey;
         public static object WindowTextKey { get; set; } = SystemColors.WindowTextBrushKey;
-        public static object WindowKey { get; set; } = SystemColors.WindowColorKey;
+        public static object WindowKey { get; set; } = SystemColors.WindowBrushKey;
```

Comment:
I think its fine to leave it as is

---
In reply to: [72653040](https://github.com/Microsoft/RTVS/pull/2122#discussion_r72653040) [](ancestors = 72653040)